db/models.py

```diff
@@ -1 +1,51 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race,
+        on_delete=models.CASCADE,
```

Comment:
Are you sure you have to delete a skill if some Race gets deleted? What if it's used by other Race?

> This is what checklist says:
> 2) Use everything for its intended purpose
> - Don't confuse the functionality of the on_delete parameter
> Use on_delete=models.CASCADE if you know that some model doesn't exist without other model. So when the referenced object is deleted, also delete the objects that have references to it.
> 
> Use on_delete=models.SET_NULL if you know that the object should not be deleted, even if the object it references may be deleted.